workflows/qc/quality-check-standard.wdl

```diff
@@ -91,6 +91,9 @@ workflow quality_check {
         call samtools.subsample as samtools_subsample { input: bam=validate_bam.validated_bam, max_retries=max_retries }
         call picard.bam_to_fastq { input: bam=samtools_subsample.sampled_bam, max_retries=max_retries }
         call fq.fqlint { input: read1=bam_to_fastq.read1, read2=bam_to_fastq.read2, max_retries=max_retries }
+        String prefix = basename(bam_to_fastq.read1, "_R1.fastq")
+        Array[File] fastqs = [bam_to_fastq.read1, bam_to_fastq.read2]
+        call ngsderive.encoding as ngsderive_encoding { input: fastqs=fastqs, prefix=prefix, max_retries=max_retries }
```

Comment:
Should this be passed to fastq_screen?

Reply:
Yes this was just an initial commit to test the encoding itself before adding the encoding parsing to fastq_screen.